doc/dev/mfx_trace2-dev.md

```diff
@@ -0,0 +1,75 @@
+# MediaSDK tracing (mfx_trace2). Developer documentation
+
+## How to use mfx_trace2?
+
+See [mfx_trace2 user documentation](./mfx_trace2.md).
+
+## Overview
+
+Main trace system object is presented by global instance of `mfx::Trace` class (instance is called `_mfx_trace`).
+Its constructor (`mfx::Trace::Trace()`) reads configuration file (on Linux) or registry values (on Windows), initializes all enabled backends.
+
+User is not supposed to touch internals of `mfx::Trace` class except `mfx::Trace::Scope` which is public interface for user to enable scoped tracing. Scope saves all events in std::vector `_mfx_trace.events` where all backends can find them using `mfx::Trace::pushEvent` function.
```

Comment:
I did not understand about pushEvent at all. Method name does not suggest that anyone can get information from it.

Reply:
In pushEvent we:
- push event to events vector
- send event to all enabled backends

Reply:
I rewrote this sentence.